api/config/swagger.py

```diff
@@ -378,7 +378,7 @@ def schema_view_swagger(request):
             'Questions': {
```

Comment:
This file has a lot of code duplication which is a necessary evil. For this reason, it should be ignored on the `.codeclimate.yml` so that code analysis is not run against it.

Reply:
Le me fix that @davenjeru

Reply:
Fixed @davenjeru